misc-tools/import-contest

```diff
@@ -212,6 +215,8 @@ if os.path.exists('problems.yaml') or os.path.exists('problems.json') or os.path
                 problems = [problem['short-name']
                             for problem in problemData['problems']]
 
+        askConfirmation = confirm("Confirm for every problem", True)
```

Comment:
nit: I think we should make this a little bit more clear, e.g.

`Confirm individually for every problem` and then set it default to `False` and rename the variable to `confirmIndividually`

Reply:
Just to make it explicit:
Set it default to False would make that we just import everything unless someone really wants the old behavior.